rmf_site_format/src/texture.rs

```diff
@@ -33,3 +33,13 @@ pub struct Texture {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub height: Option<f32>,
 }
+
+#[derive(Serialize, Deserialize, Debug, Default, Clone)]
+#[cfg_attr(feature = "bevy", derive(Bundle))]
+pub struct TextureGroup {
+    pub name: NameInSite,
+    #[serde(flatten)]
```

Comment:
I believe this would break file saving / loading if merged (as documented in https://github.com/open-rmf/rmf_site/pull/166)